.github/workflows/docsearch-crawl.yml

```diff
@@ -11,12 +11,17 @@ jobs:
     steps:
       - name: Checkout codebase
         uses: actions/checkout@v2
+      - name: Install node and ts-node
+        uses: actions/setup-node@v2
+        with:
+          node-version: "14"
+      - run: npm install -g ts-node
       - name: Crawl the site
         env:
           APPLICATION_ID: ${{ secrets.DOCSEARCH_APP_ID }}
           API_KEY: ${{ secrets.DOCSEARCH_API_KEY }}
         run: |
           docker run \
           -e APPLICATION_ID -e API_KEY \
-          -e CONFIG="$(node .github/workflows/docsearchConfigScript.js | cat .github/workflows/docsearchConfig.json)" \
+          -e CONFIG="$(ts-node -O '{\"module\": \"commonjs\"}' .github/workflows/docsearchConfigScript.js | cat .github/workflows/docsearchConfig.json)" \
```

Comment:
`ts-node` is required here just because `docsearchConfigScript.js` depends on a ts file import.

Reply:
Depends on the js/Java Script(not ts)* File being imported** 👍 👍